### DISCUSSION TITLE
I would absolutely _love_ more documentation like this:

### DISCUSSION THREAD

**Discussion by pbiggar** [darklang/dark#4360] [Feedback]
A user says:
>Honestly, it's pretty great! I'm testing it by building a Vue app that replicates Mihaly Csikszentmihalyi's test for flow (repo) and I'm using darklang as my backend. 
>
>So far, I've used the Datastore, HTTP, and REPL and I got it all hooked up with my app on the first try, which is SO impressive. It tells me the documentation you guys have is pretty great so far :) 
>
>I also LOVE the history that gets displayed on the canvas for HTTP requests. It was SO NICE in being able to see exactly what got sent. And the error message that gets displayed is AMAZINGly helpful! 
>
>I would absolutely _love_ more of[ https://ops-documentation.builtwithdark.com/](https://ops-documentation.builtwithdark.com/). Maybe an example for each case? For example, it took me a while to realize what DB::queryOne was expecting for the Block filter argument, and seeing an example would have made it obvious. 
>
>One other thing I noticed is that on my windows machine, the canvas doesn't scroll. But it all works fine on my mac. (using chrome in both cases with all my extensions turned off)

**Comment by StachuDotNet** [comment]
> I would absolutely love more of[ https://ops-documentation.builtwithdark.com/](https://ops-documentation.builtwithdark.com/). Maybe an example for each case? For example, it took me a while to realize what DB::queryOne was expecting for the Block filter argument, and seeing an example would have made it obvious.

I really like this idea of having examples within the Editor; curious how we could achieve this.
Whatever solution is settled on should apply to both the StdLib as well as packages (see [here](https://github.com/darklang/dark/discussions/4319#discussioncomment-3189851) for some related thoughts)
